### PR TITLE
Suggestion on how to do ESM builds in monorepo

### PR DESCRIPTION
**Pull request by freiksenet**
Currently it's not actually possible to use .mjs generated modules in this package repos - they will end up importing normal js. This is a hacky proposed solution to this, it annotates files with a fully qualified export.

### CODE REVIEW

**Comment by freiksenet** [comment]
I need to fix and test all other packages in the monorepo before this can be merged.

File: scripts/just.config.ts
Comment (alloy) [inline scripts/just.config.ts:54]: What's `args.importer`?

Comment (freiksenet) [reply scripts/just.config.ts:54]: This plugin runs whenever esbuild is trying to "bundle" the package. Importer checks that something is importing this package. We rewrite it to include '.js' extension then.

Comment (alloy) [reply scripts/just.config.ts:54]: Gotcha, ta

File: packages/relay-compiler-language-graphitation/tsconfig.json
Comment (alloy) [inline packages/relay-compiler-language-graphitation/tsconfig.json:3]: Hmm, don't recall why I changed this but I think there was a reason for it. I assume at least everything builds, so let's see if I run into an issue again.

**Comment by freiksenet** [comment]
@alloy No need, so I'll move it outside.

File: examples/supermassive-todomvc/.prettierrc
Comment (alloy) [inline examples/supermassive-todomvc/.prettierrc:4]: No need to block this PR on, but this can just go into the root of the repo?

File: examples/supermassive-todomvc/src/schema/resolvers.ts
Comment (alloy) [inline examples/supermassive-todomvc/src/schema/resolvers.ts:2]: Still needed?

File: package.json
Comment (alloy) [inline package.json:13]: 👍

**Review by alloy — APPROVED** [review]
:shipit:

**Pull request merged**
